pkg/commands/rules.go

```diff
@@ -117,6 +117,22 @@ func (r *RuleCommand) Register(app *kingpin.Application) {
 			Envar("CORTEX_TENANT_ID").
 			Required().
 			StringVar(&r.ClientConfig.ID)
+
+		c.Flag("tls-ca-cert", "TLS CA certificate to verify cortex API as part of mTLS, alternatively set CORTEX_TLS_CA_CERT.").
+			Default("").
+			Envar("CORTEX_TLS_CA_CERT").
+			StringVar(&r.ClientConfig.TLS.CAPath)
+
+		c.Flag("tls-cert", "TLS client certificate to authenticate with cortex API as part of mTLS, alternatively set CORTEX_TLS_CLIENT_CERT.").
+			Default("").
+			Envar("CORTEX_TLS_CLIENT_CERT").
+			StringVar(&r.ClientConfig.TLS.CertPath)
+
+		c.Flag("tls-key", "TLS client certificate private key to authenticate with cortex API as part of mTLS, alternatively set CORTEX_TLS_CLIENT_KEY.").
```

Comment:
nit: Could you name this flag `tls-key-path` to make it explicit? Similarly for the environment variable names and the cert and ca-cert flags above.